src/screens/tenants/CreateTenantDialog.tsx

```diff
@@ -46,14 +46,18 @@ export default class CreateTenantDialog extends React.Component<Props, State> {
 
   public constructor(props: Props) {
     super(props);
-    this.tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
     this.state = {
       newTenantSubDomain: null,
       newTenantName: null,
-      newTenantEndpointCloud: this.tenantEndpointClouds.find((tenantEndpointCloud) => tenantEndpointCloud.id === 'scp')
+      newTenantEndpointCloud: null
     };
   }
 
+  public componentDidMount() {
+    this.tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
+    this.setState({ newTenantEndpointCloud: this.tenantEndpointClouds.find((tenantEndpointCloud) => tenantEndpointCloud.id === 'scp') });
```

Comment:
I do not understand why the endpoints are restricted to scp here. End users shall be able to add tenants hosted on AWS.

Reply:
It's not restricted to scp, it's the initialisation of the combo box